customSDK/servicefabric/service_fabric_client_ap_is.py

```diff
@@ -9652,6 +9658,105 @@ def get_container_logs_deployed_on_node(
 
         return deserialized
 
+    def invoke_container_api(
+            self, node_name, application_id, service_manifest_name, code_package_name, code_package_instance_id, container_api_request_body, timeout=60, custom_headers=None, raw=False, **operation_config):
+        """Invoke container API on a container deployed on a Service Fabric node.
+
+        Invoke container API on a container deployed on a Service Fabric node
+        for the given code package.
+
+        :param node_name: The name of the node.
```

Comment:
Can you move the docstring documentation to the help files? More info here: https://github.com/Azure/service-fabric-cli/wiki/Command-help